### DISCUSSION TITLE
White screen after the client opens the door

### DISCUSSION THREAD

**Discussion by Speratum** [Melledy/Grasscutter#1984] [General]
Everything is fine on the server side, everything is deployed, but the door is white when the game is loaded.

**Comment by zuiwomeirenxi** [comment]
> Everything is fine on the server side, everything is deployed, but the door is white when the game is loaded.

i have this problem too.did u solve it?

**Comment by NameLord** [comment]
The heck? then why ur not opening an issue ?

**Comment by obronca** [comment]
Create new account on grasscutter cmd, and see if the issue fixed

**Reply by zuiwomeirenxi** [reply]
i did that.but it dosen't work still

**Comment by awaae001** [comment]
代理程式的问题
切换其他的代理或使用全局代理（不建议）

不要同时使用多个代理，有可能带来冲突


还有
账户的问题，你的账户出现了违禁物品（test物品）或低版本升级会带来这样的问题（小概率）

可以在CMD中使用指令
account create
创建新账号
